themes.gitconfig

```diff
@@ -384,3 +384,41 @@
     merge-conflict-theirs-diff-header-style = "#FABD2F" bold overline
     merge-conflict-ours-diff-header-decoration-style = ''
     merge-conflict-theirs-diff-header-decoration-style = ''
+
+[delta "discord-dark"]
+    # author: https://github.com/ILikePlayingGames
```

Comment:
The `dark` or `light` field is required to get the theme listed with `--show-themes`.
```suggestion
    # author: https://github.com/ILikePlayingGames
    dark                          = true
```